include/semver/semver.hpp

```diff
@@ -31,17 +31,29 @@ SOFTWARE.
 
 namespace semver
 {
+
+    using BuildIdentifierPart = uint64_t;
```

Comment:
btw., this code is using `variable_name` style, not `VariableStyle`, for naming variables, functions, and types. It's better to use the same style when modifying existing code so that code looks consistent.